e2e/cypress/support/commands.ts

```diff
@@ -53,6 +53,58 @@ Cypress.Commands.add('applyProjectsFilter', (projectsToFilterOn: string[]) => {
   }
 });
 
+Cypress.Commands.add('generateAdminToken', (idToken: string) => {
+  const adminTokenObj = {
+    id: 'cypress-api-test-admin-token',
+    name: 'cypress-api-test-admin-token'
+  };
+
+  // delete old token if exists
+  cy.request({
+    auth: { bearer: idToken },
+    method: 'DELETE',
+    url: `/apis/iam/v2beta/tokens/${adminTokenObj.id}`,
+    failOnStatusCode: false
+  }).then((resp: Cypress.ObjectLike) => {
+    expect([200, 404]).to.include(resp.status);
+  });
+
+  // create token
```

Comment:
I had to think for a minute whether it is OK to use a v2 endpoint if this might be a v1 system... so perhaps add a comment to that effect:
```
// OK to run v2 endpoints even if system is v1
```